src/lib/y2users/linux/writer.rb

```diff
@@ -216,6 +217,24 @@ def change_password(user, issues)
         set_password_attributes(user, issues)
       end
 
+      # Writes authorized keys for given user
+      #
+      # @see Yast::Users::SSHAuthorizedKeyring#write_keys
+      #
+      # @param user [Y2User::User]
+      # @param issues [Y2Issues::List] a collection for adding issues if something goes wrong
+      def write_auth_keys(user, issues)
+        return unless user.home
+
+        Yast::Users::SSHAuthorizedKeyring.new(user.home, user.authorized_keys).write_keys
+      rescue Yast::Users::SSHAuthorizedKeyring::PathError => e
+        issues << Y2Issues::Issue.new(
+          # TRANSLATORS: %s is a placeholder for a username
+          format(_("Error writing authorized keys for '%s'"), user.name)
+        )
+        log.error("Error writing authorized keys for '#{user.name}' - #{e.message}")
+      end
+
```

Comment:
NP: As I commented in another PR, this writer is growing and growing. I feel we need to restructure it with something like Action classes. But this is probably out of the scope of this PR.